app/src/models/Player.ts

```diff
@@ -0,0 +1,17 @@
+class Player {
+  constructor(
+    public name: string,
+    public score: number,
+    public color: string
+  ) {}
+
+  public static createPlayers(names: string[], colors: string[]): Player[] {
```

Comment:
This function is not needed.  You can create objects through constructor and fill players array where objects are created